Hw2.cpp

IT-kakaya opened the pull request.

IT-kakaya committed:
Create task1.cpp

IT-kakaya committed:
Create task2.cpp

IT-kakaya committed:
Add files via upload

dmitrii-kruglov commented on task2.cpp:
```diff
@@ -0,0 +1,26 @@
+    if (amount!=0){
+        cout<<"Not all numbers are positive"<<endl;
+    }
+    else cout<<"All numbers are positive"<<endl;
```
BP:  cout должен идти на следующей строке

dmitrii-kruglov commented on task2.cpp:
```diff
@@ -0,0 +1,26 @@
+    }
+    else cout<<"All numbers are positive"<<endl;
+}
+//Оно не работает так, как нужно (числа не перестают вводиться с пробелом, но если ввод закончить, то остальная часть работает, проблема только с остановкой ввода чисел), но я старалась, не придумала ничего другого
```
Окей) А блок схему не получилось нарисовать?

dmitrii-kruglov commented on task2.cpp:
```diff
@@ -0,0 +1,26 @@
+    int amount=0;
+    int a;
+    cout<<"Enter numbers. If you want to stop, enter 0:"<<endl;
+    while(char(a)!=13){
```
Зачем приводить к char если сравниваете с int?)

dmitrii-kruglov commented on task1.cpp:
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+
+int Reverse(int start){
```
BP: добавить комментарий с описанием функции

dmitrii-kruglov reviewed (approved):
Это пока наилучшее решение, что я видел) Принято (хотелось бы ещё и блок схему увидеть, но хотя бы так)

IT-kakaya committed:
Update task2.cpp

Это решение продублировано в HW3.cpp, так как было задано повторно. Но я рассчитываю на творческие баллы, потому что старательно выполняю каждое дз, поэтому делаю исправления и в ветке с HW2.cpp, в которой первоначально было это задание)

dmitrii-kruglov reviewed (approved):
Принято, без новых комментариев
